src/buildCommon.js

```diff
@@ -41,6 +41,23 @@ var makeSymbol = function(value, style, mode, color, classes) {
 };
 
 /**
+ * Makes a strut
+ */
+var makeStrut = function() {
+    return new domTree.symbolNode("", 0.85, 0.35);
```

Comment:
I'm probably being pedantic, but shouldn't this be 0.84 and 0.36? After all, we have a `\baselineskip` of `1.2em` and a strut with height `0.7\baselineskip` and depth `0.3\baselineskip`.

I also wonder: do we need to create a new node every time, or would it be all right to insert the same leaf in multiple places of the internal DOM representation?